test/cli_run_blkio_test.go

```diff
@@ -79,7 +79,7 @@ func (suite *PouchRunBlkioSuite) TestRunBlockIOWeightDevice(c *check.C) {
 
 	SkipIfFalse(c, func() bool {
 		file := fmt.Sprintf("/sys/block/%s/queue/scheduler",
-			strings.Trim(testDisk, "/dev/"))
+			strings.Trim(testDisk, "/dev"))
```

Comment:
I think here we should use `TrimPrefix` instead of `Trim`. WDYT? @ZYecho @Letty5411